### PR TITLE
Add link to charter from sig readme

### LINKED ISSUE #2243: There should be a field in sigs.yaml for location of charter
Currently the charter link is sprinkled into mission statement in many cases, while there is no first-class field for the location of a charter. We should add a first-class field for this information.

### PR DESCRIPTION
**Pull request by cblecker**
fixes #2243.

Adds first class field to sig readme for link to charter

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -229,6 +231,7 @@ sigs:
     mission_statement: >
       The Architecture SIG maintains and evolves the design principles of Kubernetes, and provides a consistent
       body of expertise necessary to ensure architectural consistency over time.
+    charter_link: chater.md
```
Comment (nikhita) [inline sigs.yaml]: s/chater/charter

File: sigs.yaml [hunk 2]
```diff
@@ -1906,6 +1937,7 @@ workinggroups:
       mailing_list: https://groups.google.com/forum/#!forum/kubernetes-wg-apply
   - name: Cloud Provider
     dir: wg-cloud-provider
+    charter_link:
```
Comment (nikhita) [inline sigs.yaml]: are working groups supposed to have a charter? https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md#sig-charter-guide says that only SIGs need to have one.

Comment (nikhita) [reply sigs.yaml]: if WGs are also supposed to have a charter, can we add the `charter_link` field for all WGs for consistency? this only adds for wg-cloud-provider.

- wg-apply has a comment which says "we might want to add a charter here".
- wg-app-def has a gdoc as their charter in the mission statement.

**Comment by cblecker** [comment]
@carolynvs @nikhita Updated. PTAL.

**Comment by nikhita** [comment]
We should also update this doc https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md to say that even WGs have to have a charter (but that can be a separate PR).

/lgtm

**Comment by nikhita** [comment]
/assign @idvoretskyi 
for approval

**Comment by cblecker** [comment]
I don't strictly know that they *have* to have one yet, but I don't mind providing a linking structure for those that do.

/approve

**Pull request merged**